### DISCUSSION TITLE
"FirebaseMessaging" plugin is not implemented on android

### DISCUSSION THREAD

**Discussion by serascandia** [capawesome-team/capacitor-firebase#110] [General]
Hello, I add the library to my Quasar project but I got this error when I try to run android studio on my device, **"FirebaseMessaging" plugin is not implemented on android**  you can guide to check what I missing?   And I have another question this library catch the message received when the app it's in background?  other libraries only works when the app it's in foreground.

Thanks

**Comment by robingenz** [comment]
Hi @serascandia, 

> "FirebaseMessaging" plugin is not implemented on android

Please have a look at the official troubleshooting guide: https://capacitorjs.com/docs/android/troubleshooting#plugin-not-implemented

> And I have another question this library catch the message received when the app it's in background?

Yes, this is supported.

**Comment by nachoalcazar** [comment]
> Hello, I add the library to my Quasar project but I got this error when I try to run android studio on my device, **"FirebaseMessaging" plugin is not implemented on android** you can guide to check what I missing? And I have another question this library catch the message received when the app it's in background? other libraries only works when the app it's in foreground.
> 
> Thanks

@serascandia, did you find the problem? I have the same issue.
